plugins/CoreHome/angularjs/siteselector/siteselector-model.service.js

```diff
@@ -119,7 +119,9 @@
         function loadInitialSites() {
             if (initialSites) {
                 model.sites = initialSites;
-                return;
+                var deferred = $q.defer();
+                deferred.resolve();
+                return deferred.promise;
```

Comment:
Without that change the site selector when creating a new user didn't work. UI tests for that were actually failing...